kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/KieServerImpl.java

```diff
@@ -159,6 +160,12 @@ protected void init(StartupStrategy startupStrategy) {
 
         KieServerState currentState = repository.load(KieServerEnvironment.getServerId());
 
+        // we sync env with state just in case for some env entries only
+        // if there is an update we just need to store it again
+        if(updateMutableEnvironmentState(currentState)) {
+            repository.store(KieServerEnvironment.getServerId(), currentState);
```

Comment:
perhaps a log could be included here to indicate that the state has been updated due to changes in the controller access, wdyt?

Reply:
added to updateConfigItemFromEnv so we can have more fine detail there.